src/pk_compiler.c

```diff
@@ -1152,6 +1155,7 @@ GrammarRule rules[] = {  // Prefix       Infix             Infix Precedence
   /* TK_ANDEQ      */   NO_RULE,
   /* TK_OREQ       */   NO_RULE,
   /* TK_XOREQ      */   NO_RULE,
+  /* TK_MODEQ      */   NO_RULE,
```

Comment:
The order of the tokens is very important here, `TK_MODEQ` comes after `TK_DIVEQ` in the `Tokens` so it should also match here.

Reply:
Makes sense to me, change has been made.